bitnami/concourse/templates/worker/statefulset.yaml

```diff
@@ -235,7 +251,11 @@ spec:
         {{- if .Values.worker.extraVolumes }}
         {{- include "common.tplvalues.render" (dict "value" .Values.worker.extraVolumes "context" $) | nindent 8 }}
         {{- end }}
-  {{- if not .Values.worker.persistence.enabled }}
+  {{- if and .Values.worker.persistence.enabled .Values.worker.persistence.existingClaim }}
+        - name: concourse-work-dir
+          persistentVolumeClaim:
+            claimName: {{ tpl .Values.worker.persistence.existingClaim $ }}
+  {{- else if not .Values.worker.persistence.enabled }}
         - name: concourse-work-dir
           emptyDir: {}
   {{- else }}
```

Comment:
Don't we want labels in the Persistent Volume Claim? 

Adding labels here will break upgrades to this version if they are using StatefulSets.

Reply:
It inherits by default the ones used you set at `sts.spec.selector.matchLabels`:
```console
$ kg pvc concourse-work-dir-concourse-worker-0 -o json | jq .metadata.labels
{
  "app.kubernetes.io/component": "concourse-worker",
  "app.kubernetes.io/instance": "concourse",
  "app.kubernetes.io/name": "concourse"
}
$ k explain sts.spec.volumeClaimTemplates.metadata.labels
KIND:     StatefulSet
VERSION:  apps/v1

FIELD:    labels <map[string]string>

DESCRIPTION:
     Map of string keys and values that can be used to organize and categorize
     (scope and select) objects. May match selectors of replication controllers
     and services. More info: http://kubernetes.io/docs/user-guide/labels
```
I don't see any reason to add them